cachelib/cachebench/cache/Cache-inl.h

```diff
@@ -397,6 +399,15 @@ typename Cache<Allocator>::ItemHandle Cache<Allocator>::insertOrReplace(
   return rv;
 }
 
+template <typename Allocator>
+void Cache<Allocator>::touchValue(const ItemHandle& it) const {
+  XDCHECK(touchValueEnabled());
+
+  auto ptr = reinterpret_cast<const uint8_t*>(getMemory(it));
+  auto sum = std::accumulate(ptr, ptr + getSize(it), 0ULL);
```

Comment:
leave a comment here that accumulate is intended to access all bytes of the item and nothing more.

Reply:
Done.